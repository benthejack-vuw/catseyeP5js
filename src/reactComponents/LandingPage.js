import React from 'react'
import { Link } from 'react-router-dom'

const LandingPage = () => (
    <div>

        <Link to='/pattern-builder/'>
            <p>Build a pattern</p>
        </Link>

        <Link to='/grid-builder/'>
            <p>Build a grid</p>
        </Link>
    
    </div>
)

export default LandingPage;