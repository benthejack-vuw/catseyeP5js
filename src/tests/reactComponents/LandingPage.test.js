import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../../reactComponents/LandingPage';

test('should render LandingPage correctly', ()=>{
    const wrapper = shallow(<LandingPage startLogout={()=>{}}/>);
    expect(wrapper).toMatchSnapshot();
})