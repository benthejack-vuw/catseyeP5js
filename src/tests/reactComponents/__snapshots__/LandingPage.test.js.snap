// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LandingPage correctly 1`] = `
<div>
  <Link
    to="/pattern-builder/"
  >
    <p>
      Build a pattern
    </p>
  </Link>
  <Link
    to="/grid-builder/"
  >
    <p>
      Build a grid
    </p>
  </Link>
</div>
`;
